### ISSUE TITLE
losses: Not able to load Huberloss 

### ISSUE THREAD

**Issue by iamyihwa** [Nixtla/neuralforecast#649]
### What happened + What you expected to happen

Not able to load HuberLoss

![image](https://github.com/Nixtla/neuralforecast/assets/12494156/d86cff26-6a53-4528-9240-1c20cffbe139)

Other loss functions I could load fine .. 
![image](https://github.com/Nixtla/neuralforecast/assets/12494156/c7f68808-95c3-4032-9d9a-83d5f77adbaf)

### Versions / Dependencies

1.5.0 

### Reproduction script

from neuralforecast.losses.pytorch import  HuberLoss

### Issue Severity

None

**Comment by kdgutier** [comment]
Hey @iamyihwa,

Robust regression losses, including `HuberLoss` and the inclusion of `dropout_prob_theta` in the `NHITS` model, are available in the main branch:

To install the main branch, you can use this line:
```
!pip --upgrade install git+https://github.com/Nixtla/neuralforecast.git
```

Here is the robust forecasting tutorial: https://nixtla.github.io/neuralforecast/examples/robust_regression.html

**Comment by cchallu** [comment]
The new release is public! These new functions are already available in the `pip` version.
